app/models/reviewable_queued_post.rb

```diff
@@ -72,6 +72,23 @@ def build_legacy_combined_actions(actions, guardian, args)
   end
 
   def build_new_separated_actions(actions, guardian, args)
+    # Because a queued post isn't a real post, we need to create our own post actions bundle
```

Comment:
Ah, yes. This has come to bite me before. Fixing this would take a lot of rewiring, so let's just do this for now. 👍